eox_tenant/models.py

```diff
@@ -65,6 +66,59 @@ def get_microsite_for_domain(cls, domain):
         return microsites[0] if microsites else None
 
 
+class TenantConfig(models.Model):
+    """
+    Model to persist edxapp configurations.
+    """
+    key = models.CharField(max_length=63, db_index=True)
+    values = JSONField(null=False, blank=True, load_kwargs={'object_pairs_hook': collections.OrderedDict})
+    lms_configs = JSONField(null=False, blank=True, load_kwargs={'object_pairs_hook': collections.OrderedDict})
+    studio_configs = JSONField(null=False, blank=True, load_kwargs={'object_pairs_hook': collections.OrderedDict})
+    theming_configs = JSONField(null=False, blank=True, load_kwargs={'object_pairs_hook': collections.OrderedDict})
+    sites = models.ManyToManyField(Site)
```

Comment:
Why is this a ManyToMany? one site (domain) will not have many configurations. Having a many to many requires one extra lookup in a tenant-sites table instead of placing the foreign_key of the site in this table.